File: python/quickstart.py
"""
CSMarketAPI Quickstart Example

Get real-time CS2 skin prices across multiple marketplaces.
"""
import asyncio
import os
from csmarketapi import CSMarketAPI
from csmarketapi.enums import Market, Currency


async def main():
    # Get API key from environment variable
    # Set it with: export CSMARKETAPI_KEY="your_api_key"
    async with CSMarketAPI(os.environ["CSMARKETAPI_KEY"]) as client:
        res = await client.get_listings_latest_aggregated(
            market_hash_name="Chroma 2 Case",  # Replace with your item
            markets=list(Market),
            currency=Currency.USD
        )

        print(f"Item: {res.market_hash_name}")
        for listing in res.listings:
            print(f"  {listing.market}: ${listing.min_price:.2f}")


if __name__ == "__main__":
    asyncio.run(main())

File: javascript/quickstart.js
/**
 * CSMarketAPI Quickstart Example
 *
 * Get real-time CS2 skin prices across multiple marketplaces.
 */
import { CSMarketAPI, Currency } from "csmarketapi"

// Get API key from environment variable
// Set it with: export CSMARKETAPI_KEY="your_api_key"
const client = new CSMarketAPI({
  apiKey: process.env.CSMARKETAPI_KEY
})

const res = await client.getListingsLatestAggregated({
  marketHashName: "Chroma 2 Case",  // Replace with your item
  currency: Currency.USD,
})

console.log(`Item: ${res.market_hash_name}`)
for (const listing of res.listings) {
  console.log(`  ${listing.market}: $${listing.min_price.toFixed(2)}`)
}
